### PR TITLE
first commit audit

### PR DESCRIPTION
**Pull request by jcriotte**

### CODE REVIEW

File: src/Controller/AuditController.php [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            $entityManager->flush();
+
+            return $this->redirectToRoute('audit_index', [], Response::HTTP_SEE_OTHER);
+        }
+
+        return $this->renderForm('audit/test.html.twig', [
+            'audit' => $audit,
+            'form' => $form,
+        ]);
+    }
```
Comment (aldaitzwild) [inline src/Controller/AuditController.php:114]: What the purpose of this route ?

File: templates/audit/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+				<tbody>
+					{% for audit in audits %}
+						<tr>
+							<td>{{ audit.id }}</td>
+							<td>{{ audit.name }}</td>
+							<td>{{ audit.datecreation ? audit.datecreation|date('Y-m-d') : '' }}</td>
+							<td>
+								<a href="{{ path('audit_show', {'id': audit.id}) }}">show</a>
+								<a href="{{ path('audit_edit', {'id': audit.id}) }}">edit</a>
+								<a href="{{ path('audit_test', {'id': audit.id}) }}">Lancer un test</a>
```
Comment (aldaitzwild) [inline templates/audit/index.html.twig]: French or English ! Not both.

Comment (aldaitzwild) [reply templates/audit/index.html.twig]: Check All files because you the same problems on various templates

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Comment by jcriotte** [comment]
New version after correction

**Comment by jcriotte** [comment]
My GIT PUSH didn’t work

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**